cli/cli.rb

```diff
@@ -65,6 +67,20 @@ def get_port()
       puts "Port #{port} is free to use!"
     end
 
+    desc "resources", "See your Nest resource usage and limits"
+    def resources()
+      diskUsage = (Float(run("quota  | tail -n 1 | awk '{print $2}'", { capture: true, verbose: false })) / (1024 * 1024)).round(2)
```

Comment:
Does the `quota` command need to run twice? Also, could the string processing run natively in ruby somehow?